src/node_errors.h

```diff
@@ -102,6 +114,18 @@ void PrintErrorString(const char* format, ...);
   V(ERR_TRANSFERRING_EXTERNALIZED_SHAREDARRAYBUFFER,                         \
     "Cannot serialize externalized SharedArrayBuffer")                       \
 
+#if HAVE_OPENSSL && defined(NODE_EXPERIMENTAL_QUIC)
+# define QUIC_PREDEFINED_ERROR_MESSAGES(V)                                   \
+  V(ERR_QUIC_CANNOT_SET_GROUPS, "Cannot set groups")                         \
+  V(ERR_QUIC_FAILURE_SETTING_SNI_CONTEXT, "Failure setting SNI context")
+#else
+# define QUIC_PREDEFINED_ERROR_MESSAGES(V)
+#endif
+
+#define PREDEFINED_ERROR_MESSAGES(V)                                         \
+  BASE_PREDEFINED_ERROR_MESSAGES(V)                                          \
+  QUIC_PREDEFINED_ERROR_MESSAGES(V)
```

Comment:
Tbh, I wouldn’t use conditionals here – everything that happens here happens in inline functions, so if the errors aren’t used, there’s zero overhead.